esp-hal-common/src/soc/esp32/radio_clocks.rs

```diff
@@ -1,3 +1,17 @@
+//! Radio clocks driver(ESP32)
+//!
+//! ## Overview
+//!
+//! The `Radio Clocks` module is a part of the `SOC` functionality of the
```

Comment:
What is this sentence trying to say? As a user, reading "SOC functionality" confuses me. This is MCU-specific, but I don't think that is necessary to be pointed out in a MCU-specific set of documentation.